cluster/gce/config-default.sh

```diff
@@ -140,6 +140,10 @@ export AGGREGATOR_MASTER_NAME="${INSTANCE_PREFIX}-aggregator"
 export INITIAL_ETCD_CLUSTER="${MASTER_NAME}"
 export MASTER_TAG="${INSTANCE_PREFIX}-master"
 export NODE_TAG="${INSTANCE_PREFIX}-minion"
+# ETCD_LISTEN_CLIENT_IP decides whether etcd servers should also listen on host IP, 
+# in addition to listening to 127.0.0.1., and whether kube-apiserver should connect to etcd servers
+# through host IP.
+export ETCD_LISTEN_CLIENT_IP="${ETCD_LISTEN_CLIENT_IP:-false}"
```

Comment:
The way how it is initialized seems wrong given its later usage:
https://github.com/kubernetes/kubernetes/pull/95312/files#diff-ada96bec4aba13195248d6dd21ee67e333bead2bd898b81a1b089d53bd06b4d8R1738

Reply:
I overlooked a naming conflict. Thanks for catching it!

Reply:
Fixed the variable name. Shoult be ETCD_LISTEN_HOST_IP